### PR TITLE
Advisory data collection fix

### PR DESCRIPTION
**Pull request by ashwindasr**
The dashboard was initially getting data from `group.yml`. Changed to `releases.yml`. The advisories are sorted by `brew_event` in decreasing order. The advisory with the highest `brew_event` value is considered as the newest advisory and returned to front end.

### CODE REVIEW

File: lib/http_requests.py [hunk 1]
```diff
@@ -13,17 +13,13 @@
...
     """
     This function lists all the branches of the ocp-build-data repository.
     :return: dict, all the branches along with their details.
     """
 
-    access_token = os.environ["GITHUB_PERSONAL_ACCESS_TOKEN"]
-
     try:
-        req = requests.get(app_constants.GITHUB_URL_TO_LIST_ALL_OCP_BUILD_DATA_BRANCHES,
-                           headers={"Authorization": "token " + access_token})
```
Comment (sosiouxme) [inline lib/http_requests.py:26]: why remove auth?

yes, it's an unauthenticated endpoint, but i think this was intended to solve rate-limiting problems. our token has a way higher rate-limit than you get without it.

File: lib/http_requests.py [hunk 2]
```diff
@@ -52,128 +48,74 @@ def get_all_ocp_build_data_branches():
...
 def get_group_yml_file_url(branch_name: str) -> dict:
-
     """
     This method takes a branch name of ocp_build_data as a parameter and returns advisories details for the same.
     :param branch_name: Branch name of ocp_build_data
     :return: adivsories details
     """
-    access_token = os.environ["GITHUB_PERSONAL_ACCESS_TOKEN"]
     hit_url = app_constants.GITHUB_GROUP_YML_CONTENTS_URL.format(branch_name)
-    hit_request = requests.get(hit_url, headers={"Authorization": "token " + access_token})
```
Comment (sosiouxme) [inline lib/http_requests.py:69]: likewise

File: lib/http_requests.py [hunk 3]
```diff
@@ -52,128 +48,74 @@ def get_all_ocp_build_data_branches():
...
+    advisory_data = []
+    for version in yml_data:
+        try:
+            advisories = yml_data[version]['assembly']['group']['advisories']
+            brew_event = yml_data[version]['assembly']['basis']['brew_event']
+        except:
+            continue
+        # advisory_data += list(advisories)
+        if -1 in advisories or 1 in advisories:
+            continue
```
Comment (sosiouxme) [inline lib/http_requests.py:98]: `-1` or `1` will not be keys in the advisories, they'll be values, so...
```suggestion
        if -1 in advisories or 1 in advisories.values():
            continue
```

File: ocp_build_data/migrations/0001_initial.py [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Generated by Django 3.0.7 on 2020-07-30 04:44
+# Generated by Django 3.0.7 on 2022-08-02 15:36
```
Comment (sosiouxme) [inline ocp_build_data/migrations/0001_initial.py:1]: hmm... yes... why not add another migration instead of starting over?

**Comment by ashwindasr** [comment]
Added github token and removed database logic.

**Comment by ashwindasr** [comment]
The reason why the env variables from `prod.env` and `dev.env` didn't get set could be because there were spaces. Can try removing the ones we manually defined in the pod, that are common with the `prod.env` and see if it works.

**Review by sosiouxme — APPROVED** [review]
:shipit:

**Pull request merged**